src/Rules/Functions/FunctionCallableArgExistsRule.php

```diff
@@ -0,0 +1,93 @@
+<?php declare(strict_types = 1);
+
+namespace PHPStan\Rules\Functions;
+
+use PhpParser\Node;
+use PhpParser\Node\Expr\FuncCall;
+use PHPStan\Analyser\Scope;
+use PHPStan\Broker\Broker;
+use PHPStan\Rules\CallableExistsCheck;
+use PHPStan\Type\CallableType;
+
+class FunctionCallableArgExistsRule implements \PHPStan\Rules\Rule
+{
+
+	/**
+	 * @var \PHPStan\Broker\Broker
+	 */
+	private $broker;
+
+	/**
+	 * @var \PHPStan\Rules\CallableExistsCheck
+	 */
+	private $check;
+
+	public function __construct(
+		Broker $broker,
+		CallableExistsCheck $check
+	)
+	{
+		$this->broker = $broker;
+		$this->check = $check;
+	}
+
+	public function getNodeType(): string
+	{
+		return FuncCall::class;
+	}
+
+	/**
+	 * @param \PhpParser\Node\Expr\FuncCall $node
+	 * @param \PHPStan\Analyser\Scope $scope
+	 * @return string[]
+	 */
+	public function processNode(Node $node, Scope $scope): array
+	{
+		if (!($node->name instanceof \PhpParser\Node\Name)) {
+			return [];
+		}
+
+		if (!$this->broker->hasFunction($node->name, $scope)) {
+			return [];
+		}
+
+		$function = $this->broker->getFunction($node->name, $scope);
+		$parameters = $function->getParameters();
+
+		$errors = [];
+
+		foreach ($node->args as $i => $argument) {
+			if (!$parameters[$i]->getType() instanceof CallableType &&
+				!$this->isCallableArgOfCoreFunction($function->getName(), $i + 1)) {
+				continue;
+			}
+
+			$msgPrefix = sprintf('Argument #%d of %s should be callable, but passed ', $i + 1, $node->name);
```

Comment:
The name of the argument is also added, for example: `'Parameter #%d %s of method ' . $messagesMethodName . ' expects %s, %s given.',` (see usages of FunctionCallParametersCheck).